astroplan/core.py

```diff
@@ -296,21 +366,40 @@ def ra(self):
         """
         Right ascension.
         """
-        raise NotImplementedError
+        if isinstance(self, FixedTarget):
+            return self.coord.ra
+        raise NotImplementedError()
 
     @property
     def dec(self):
         """
         Declination.
         """
-        raise NotImplementedError
+        if isinstance(self, FixedTarget):
+            return self.coord.dec
+        raise NotImplementedError()
 
 
 class FixedTarget(Target):
     """
     An object that is "fixed" with respect to the celestial sphere.
     """
-
+    def __init__(self, coord, **kwargs):
+        '''
+        TODO: Docstring.
+        '''
+        if not isinstance(coord, SkyCoord):
```

Comment:
This should probably accept a `SkyCoord` or any of the `BaseCoordinateFrame` classes? (e.g., `ICRS`)